app/modules/users/resources.py

```diff
@@ -83,6 +83,7 @@ def post(self, args):
             abort(code=HTTPStatus.BAD_REQUEST, message='Must provide a password')
 
         args['is_active'] = True
+        args['in_alpha'] = True
```

Comment:
I understood that "alpha" meant "User has access to alpha release of product" so it seems wrong that this is being proposed as what any user has. 
I'd suggest asking @naknomum if this is the correct fix.

Reply:
yes, it seems to me _long-term_ we would want to: (a) modify login to allow in _some additional_ role of user(`active`?) (b) modify this part of the code to instead add _that_ role for admin-created users.

however, until we get some clarity on _that_, i dont see a huge problem with this as a sort of stop-gap measure.  alpha... or beta?   unless we wanna try to make the above decision before moving on with this.  maybe a good friday topic!

Reply:
Surely the better fix is to change the auth code to allow any active user to log in. 
That's what active means isn't it?